Simplenote/NSMutableAttributedString+Styling.m

```diff
@@ -1,62 +1,66 @@
-//
-//  NSMutableAttributedString+TruncateToWidth.m
-//  Simplenote
-//
-//  Created by Rainieri Ventura on 4/5/12.
-//  Copyright (c) 2012 Simperium. All rights reserved.
-//
-
 #import "NSMutableAttributedString+Styling.h"
+#import "NSString+Bullets.h"
 #import "Simplenote-Swift.h"
-#import "SPTextView.h"
 
-@implementation NSMutableAttributedString (Styling)
 
-const int RegexExpectedMatchGroups  = 3;
-const int RegexGroupIndexPrefix     = 1;
-const int RegexGroupIndexContent    = 2;
 
-// Replaces checklist markdown syntax with SPTextAttachment images in an attributed string
-- (void)addChecklistAttachmentsForColor: (UIColor *)color  {
+@implementation NSMutableAttributedString (Checklists)
 
+- (void)processChecklistsWithColor:(UIColor *)color
+                        sizingFont:(UIFont *)sizingFont
+             allowsMultiplePerLine:(BOOL)allowsMultiplePerLine
+{
     if (self.length == 0) {
         return;
     }
-    
-    NSError *error;
-    NSRegularExpression *regex = [NSRegularExpression regularExpressionWithPattern:CheckListRegExPattern options:NSRegularExpressionAnchorsMatchLines error:&error];
-    
-    // Work with a copy of the NSString value so we can calculate the correct indices
-    NSString *noteString = self.string.copy;
-    NSArray *matches = [regex matchesInString:noteString options:0 range:[noteString rangeOfString:noteString]];
-    
-    if (matches.count == 0) {
-        return;
-    }
-    
-    int positionAdjustment = 0;
-    CGFloat fontSize = [UIFont preferredFontForTextStyle:UIFontTextStyleHeadline].pointSize + 4;
+
+    NSString *plainString = self.string.copy;
```

Comment:
If I'm being nitpicky, I'd say since `copy` is a method and not a property it should be refereneced as `[self.string copy]` rather than use dot notation.  I'm not sure I'm being that picky tho ;)

Reply:
Swift syntax seeping into objc code style. :)  I'm noticing lots of methods called via dot notation in other places too.